docs/source/enterprise/overview.rst

```diff
@@ -1,31 +1,31 @@
-.. _teams-overview:
+.. _enterprise-overview:
 
-FiftyOne Teams Overview
-=======================
+FiftyOne Enterprise Overview
+============================
 
 .. default-role:: code
 
-FiftyOne Teams is purpose-built to integrate into your existing ML workflows,
+FiftyOne Enterprise is purpose-built to integrate into your existing ML workflows,
 including annotation, evaluation, model training, and deployment.
 
 .. note::
 
-    `Learn more <https://voxel51.com/fiftyone-teams>`_ about FiftyOne Teams and
-    `contact us <https://voxel51.com/get-fiftyone-teams>`_ to try it!
+    `Learn more <https://voxel51.com/enterprise>`_ about FiftyOne Enterprise and
+    `contact us <https://voxel51.com/talk-to-sales>`_ to try it!
 
-.. _fiftyone-vs-fiftyone-teams:
+.. _fiftyone-vs-fiftyone-enterprise:
 
-FiftyOne vs FiftyOne Teams
-__________________________
+FiftyOne vs FiftyOne Enterprise
```

Comment:
In marketing we've started using `FiftyOne Open Source` when juxtaposing with `FiftyOne Enterprise`. So for this page, can we replace `FiftyOne` and `open source FiftyOne` instances with `FiftyOne Open Source`?

One possible exception: might be cleaner to use `Open Source vs Enterprise` for the section heading here.